assets/js/components/User/login.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div class="login">
+        <label>Email:</label>
+        <input v-model="email"/>
+        <button @click.prevent="login">Login</button>
+        <label>AuthToken:</label>
```

Comment:
Why do we need the AuthToken here?

Reply:
We don't, pretty much none of the UI I have made is meant to be permanent or even used, I am just adding mocks so that I can test endpoints and make sure that the backend is behaving.